doc/input-output-reference/src/overview/group-radiative-convective-units.tex

```diff
@@ -2063,3 +2069,5 @@ \subsubsection{Inputs}\label{inputs-11-013}
       Z5VentSlabIn,            !- Slab In Node Name for Surface 3
       Z5VentSlabOut;           !- Slab Outlet Node Name for Surface 3
 \end{lstlisting}
+\]
+\end{document}
```

Comment:
@mjwitte @Myoldmopar Well, again as with another one, I'm not putting these extra things in so there must be something with the software I am using that is putting this stuff in without my knowledge.  I am using LaTeXiT on the Mac.  Any ideas why this is happening?